studio/__main_unit__.py

```diff
@@ -118,11 +135,37 @@ async def index(request: Request):
     return await root(request)
 
 
+async def check_load_balancer() -> Dict[str, Any]:
+    server_host = os.getenv("AWS_SERVER_HOST", "localhost")
+    logger.info(f"Attempting to connect to load balancer at: http://{server_host}")
+
+    try:
+        async with aiohttp.ClientSession() as session:
+            async with session.get(f"http://{server_host}", timeout=5) as response:
```

Comment:
In the production environment, it will be `https` instead of `http`.
Considering the above, it seems reasonable to specify `AWS_SERVER_HOST` in the form of a URL.
Also, a variable name such as `AWS_SERVICE_URL` is considered appropriate.